.circleci/config.yml

```diff
@@ -47,9 +47,35 @@ jobs:
       - run: npm run dist
       - run: bash <(curl -s https://codecov.io/bash) -F frontend
 
+  nightly-build:
```

Comment:
It might be good to remove this file's changes from the PR as this is unrelated to the actual documentation changes.